<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileInformationWindow</class>
 <widget class="QWidget" name="FileInformationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_filePathValue">
      <property name="text">
       <string>empty</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_vertexCount">
      <property name="minimumSize">
       <size>
        <width>98</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Vertex count:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_filePath">
      <property name="text">
       <string>FilePath:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_edgeCountValue">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>empty</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_vertexCountValue">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>empty</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_edgeCount">
      <property name="minimumSize">
       <size>
        <width>98</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Edge count:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>100</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
                             background-color: rgb(75, 75, 75); /* Background color */ 
                             color: #FFF; /* Text color */ 
                             border: 0px solid #007ACC; /* Border color and width */ 
                             border-radius: 16px; /* Rounded corners */ 
                             padding: 1px; /* Padding inside the button */ 
                         /*font-size: 11pt; */
                         } 
                          
                         QPushButton:hover { 
                             background-color: rgb(85, 85, 85); /* Background color on hover */ 
                             border: 1px solid rgb(147,112,219); /* Border color on hover */ 
                         } 
                           
                         QPushButton:pressed { 
                             background-color: rgb(65, 65, 65); /* Background color when pressed */ 
                             border: 1px solid rgb(147,112,219); /* Border color on hover */ 
                         }</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
